examples/ucp_hello_world.c

```diff
@@ -67,18 +67,25 @@ enum ucp_test_mode_t {
     TEST_MODE_EVENTFD
 } ucp_test_mode = TEST_MODE_PROBE;
 
+typedef enum {
+    FAILURE_MODE_NONE,
+    FAILURE_MODE_SEND,
```

Comment:
you are killing either server or client, i'd better reflect it in the names instead.
SEND or RECV sounds confusing

Reply:
failure mode specify what operation is failed: send/recv/keepalive

Reply:
add comments : /* fail send operation on server */
/* fail receive operation on client */